packages/workflow/src/Interfaces.ts

```diff
@@ -1012,6 +1039,23 @@ export interface INodePropertyCollection {
 	values: INodeProperties[];
 }
 
+export interface INodePropertyValueExtractorBase {
+	type: string;
+}
```

Comment:
Why do we split off the base? Is this reused in a later PR?

Reply:
This is left over from an earlier iteration. It can be removed.